src/test/kotlin/org/neo4j/graphql/TranslatorExceptionTests.kt

```diff
@@ -16,16 +16,26 @@ class TranslatorExceptionTests : AsciiDocTestSuite("translator-tests1.adoc") {
     }
 
     override fun schemaTestFactory(schema: String): List<DynamicNode> {
-        val translator = Translator(SchemaBuilder.buildSchema(schema));
+        val translator = Translator(SchemaBuilder.buildSchema(schema))
         return listOf(
                 DynamicTest.dynamicTest("unknownType") {
                     Assertions.assertThrows(IllegalArgumentException::class.java) {
-                        translator.translate(" { company { name } } ")
+                        translator.translate("""
```

Comment:
these changes can be reverted, since there was no issue with singel line graphql but with nbsp-characters